README.md

```diff
@@ -2,30 +2,45 @@
 
 A React Native Camera and Player for MediaPipe applications.
 
-## Installation
+* [Documentation](https://cdiddy77.github.io/react-native-mediapipe/)
 
+## Requirements
+* Requirements go here.
+
+## Features
+* 🎥 Video streaming from a react-native app to a media pipe
+* 🔍 AI Object Detection
+* 🧩 Works as a react-native component
+
+Install mediapipe from npm:
```

Comment:
should be: Install react-native-mediapipe from npm,

Reply:
Fixed the name